src/Resources/views/Admin/Channel/_standardForm.html.twig

```diff
@@ -80,5 +80,10 @@
             {{ form_row(form.skippingPaymentStepAllowed) }}
             {{ form_row(form.accountVerificationRequired) }}
         </div>
+        <div class="ui attached segment">
+            {{ form_row(form.channelPriceHistoryConfig.lowestPriceForDiscountedProductsVisible) }}
+            {{ form_row(form.channelPriceHistoryConfig.lowestPriceForDiscountedProductsCheckingPeriod) }}
+            {{ form_row(form.channelPriceHistoryConfig.taxonsExcludedFromShowingLowestPrice, {'remote_url': path('sylius_admin_ajax_taxon_by_name_phrase'), 'load_edit_url': path('sylius_admin_ajax_taxon_by_code')}) }}
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Check `form.channelPriceHistoryConfig` is defined before